src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const reverseKeys = {};
```

Comment:
```suggestion
  const reversedRobot = {};
```
`obj !== keys` and key**S** sound like an array